xds/src/main/java/io/grpc/xds/internal/sds/XdsServerBuilder.java

```diff
@@ -120,6 +124,12 @@ public XdsServerBuilder intercept(ServerInterceptor interceptor) {
     return this;
   }
 
+  public XdsServerBuilder fallbackProtocolNegotiator(
+      ProtocolNegotiator fallbackProtocolNegotiator) {
+    this.fallbackProtocolNegotiator = fallbackProtocolNegotiator;
```

Comment:
null check here would be nice.

Reply:
Hmmm, on second thoughts since we are making `fallbackProtocolNegotiator` optional (in the builder) if somebody wants to "nullify" the previously set value, shouldn't we allow null here?

Reply:
that's little bit subjective. but setting and unsetting on the same builder is weird. calling this has intention of setting Fallback PN, so it is okay to not accept null imo.

Reply:
But I can call it multiple times with different values - the last value wins. It doesn't make sense to disallow null in that case. If I need to clear the previously set value in a different piece of code (over which I have no control) then the only thing I can do is to call this with null assuming it is allowed.

Reply:
i don't really care which way you use because both has pros and cons. iirc, this topic was mentioned in the api meeting briefly.
make sure this is described in the javadoc.

Reply:
We normally allow people to "undo" changes to the builder, so if null is okay, allowing null is fine. But I'd just remind you not to send too much effort on this part of the API because it will be replaced.

Reply:
`null` is okay so I am going to allow null to override a previous non-null value (and mention so in javadocs)